app/src/main/res/values/strings.xml

```diff
@@ -192,6 +192,7 @@
     <string name="delete_search_history_alert">Delete entire search history?</string>
     <string name="search_history_deleted">Search history deleted.</string>
     <string name="help">Help</string>
+    <string name="item_in_history">In History</string>
```

Comment:
"Seen" would be use up less space making the thumbnail more visible. Also, while "in history" could be interpreted as "this was already encountered", "seen" better conveys the meaning that "the video was opened in the player before. At first I thought about "watched", but that means more "this video was watched thoroughly".
```suggestion
    <string name="item_in_history">Seen</string>
```

Reply:
"Seen" looks strange to me, maybe because I never seen an app that uses this term for that. I still think "In History" better represents what actually is being checked.

@TobiGr @wb9688: What do you think?

Reply:
I agree that "Seen" is weird, and I prefer e.g. "In History".